README.md

```diff
@@ -121,6 +121,17 @@ directory as the `pa_matchbatch.sh` script.
 `OVERRIDE_PATH` indicates where to find version overrides. See next section. It defaults to
 `/tmp/version_overrides.txt`.
 
+`IGNORE_REPOS` contains a (space-separated) list of repos that we do not want to process, i.e.
```

Comment:
Useful for when I've already checked a component, and do not want it cluttering the output.

Reply:
Could maybe change this to IGNORE? IGNORE_REPOS might seem a bit redundant.